Sources/Danger/GitHubDSL.swift

```diff
@@ -483,38 +483,38 @@ public struct GitHubMilestone: Decodable {
     // MARK: - Properties
 
     /// The id number of this milestone
-    let id: Int
+    public let id: Int
```

Comment:
Good catch 😱 Can't believe we missed these

Reply:
It's easy to mix up 😉

Reply:
Yeah, Swift's accessor stuff is a pain